vehicle/OVMS.V3/components/vehicle_mgev/src/mg_poll_bms.cpp

```diff
@@ -162,10 +162,29 @@ void OvmsVehicleMgEv::IncomingBmsPoll(
             break;
         case batterySoCPid:
             {
-                auto soc = value / 10.0;
+                // Get selection from features page
+                bool updatedbmu = MyConfig.GetParamValueBool("xmg", "updatedbmu", true);
```

Comment:
Might be useful to pull this out to a separate function since the MG5 also has a different range.